connectors/sources/onedrive.py

```diff
@@ -303,12 +301,19 @@ async def paginated_api_call(self, url_name, **url_kwargs):
                 break
 
     async def list_users(self):
-        async for response in self.paginated_api_call(url_name=USERS):
+        async for response in self.paginated_api_call(
+            url_name=USERS, page_size=FETCH_SIZE
+        ):
             for user_detail in response:
                 yield user_detail
 
     async def get_owned_files(self, user_id, skipped_extensions=None, pattern=""):
-        async for response in self.paginated_api_call(url_name=DELTA, user_id=user_id):
+        async for response in self.paginated_api_call(
+            url_name=DELTA,
+            user_id=user_id,
+            page_size=FETCH_SIZE,
+            item_fields=ITEM_FIELDS,
```

Comment:
We should probably make `paginated_api_call` as general as possible.

From the name `paginated_api_call`, I'm only expecting parameters like `url` and `page_size`.